src/components/global/carouselWrapper/carouselWrapper.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+import React, { FC, ReactNode } from "react";
+import Slider, { Settings } from "react-slick";
+
+import * as styled from "./carouselWrapper.styles";
+import "slick-carousel/slick/slick.css";
+import "slick-carousel/slick/slick-theme.css";
+
+export interface ICarouselSliderComp {
+  children: ReactNode;
+  settings: Settings;
+}
+
+export const CarouselWrapper: FC<ICarouselSliderComp> = ({
+  children,
+  settings,
+}) => {
+  return (
+    <styled.CarouselSlider>
+      <Slider
+        {...settings}
```

Comment:
Omdat dit een wrapper component is, is het handig om settings als arg mee te geven. Zo kunt ge voor verschillende carousels verschillende settings meegeven.